packages/neon_framework/lib/src/testing/mocks.dart

```diff
@@ -23,10 +22,17 @@ import 'package:shared_preferences_platform_interface/shared_preferences_platfor
 // ignore: depend_on_referenced_packages
 import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
 
-class MockAccount extends Mock implements Account {
-  @internal
-  @override
-  Uri completeUri(Uri uri) => uri;
+Account MockAccount({
+  Uri? serverURL,
+  String? username,
+  String? password,
+}) {
+  return Account(
+    (b) => b
+      ..serverURL = serverURL ?? Uri.parse('https://cloud.example.com:8443/nextcloud')
+      ..username = username ?? 'username'
+      ..password = password ?? 'password',
+  );
```

Comment:
Please add the default username and password value to the parameter list.

I'm also not a huge fan of this common code. I can see that you don't want to duplicate it over and over again, but I prefer unit tests to be independent.

Reply:
I think this is better because it ensures we always use a URL that is special.
This doesn't make tests depending on each other and I don't see a downside in this shared mock account.